driver/bpf/fillers.h

```diff
@@ -2248,6 +2256,29 @@ static __always_inline bool get_exe_writable(struct task_struct *task)
 	return false;
 }
 
+static __always_inline bool get_exe_upper_layer(struct inode *inode)
+{
+	struct super_block *sb = _READ(inode->i_sb);
```

Comment:
What about checking if `sb != NULL` here?
Same in the above usage inside `get_exe_writable` :)